src/deploy-cromwell-on-azure/Deployer.cs

```diff
@@ -3182,6 +3187,82 @@ private static string GetLinuxParentPath(string path)
             return string.Join(dirSeparator, pathComponents.Take(pathComponents.Length - 1));
         }
 
+        public async Task<List<MountableContainer>> GetContainersToMount(string path)
+        {
+            if (string.IsNullOrWhiteSpace(path))
+            {
+                return null;
+            }
+
+            var containers = new HashSet<MountableContainer>();
+            var exclusion = new HashSet<MountableContainer>();
+            var wildCardAccounts = new List<string>();
+            var contents = await File.ReadAllTextAsync(path);
```

Comment:
I'd recomend `File.ReadAllLinesAsync` instead. Reduces issues related to line endings since the deployer runs on multiple platforms,